core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreMainActivity.kt

```diff
@@ -54,8 +59,28 @@ abstract class CoreMainActivity : BaseActivity(), WebViewProvider {
   abstract val drawerNavView: NavigationView
   abstract val bookmarksFragmentResId: Int
   abstract val historyFragmentResId: Int
+  abstract val initialDestinationFragmentId: Int
   abstract val cachedComponent: CoreActivityComponent
 
+  override fun onCreate(savedInstanceState: Bundle?) {
+    setTheme(R.style.Base_KiwixTheme)
```

Comment:
Base_KiwixTheme? I would guess this breaks dark mode

Reply:
Surprisingly it does not break drak mode.

Reply:
still KiwixTheme would be more accurate

Reply:
This changed all our colors, KiwixTheme must be used

Reply:
Oh, that's why things are purple!